### PR TITLE
Remarks about Scheduled Execution

### PR DESCRIPTION
**Pull request by klausschuch**
As discussed in a recent design meeting and in issue #1543, this is a draft pr with comments about the current description of Scheduled Execution.

### CODE REVIEW

**Comment by MBlesken** [comment]
Okay, I made it through and adjusted SE descriptions all over the specification. It think it makes sense that @andreas-junghanns takes a look at it now. I would like to be involved in deciding if this PR is ready to be merged. But before that I expect to discuss some of the parts. I also left some questions. FYI @PTaeuberDS @klausschuch

**Comment by andreas-junghanns** [comment]
I added a few more fixes to language and formatting - please check.
Next, I think, find a slot for a meeting and go through the remaining TODOs.

**Comment by MBlesken** [comment]
**------------ Ready for Review -------------------**

The PR is now ready to be reviewed: @klausschuch could you please remove the DRAFT tag? I couldn't find how to do it and propably you are the only one allowed to do it.

Start the review: @klausschuch and @TorstenBlochwitz I would like to invite you and everyone else to leave additional comments. I'm proposing a **deadline for November 17th 2021** before we merge this PR.

FYI: @PTaeuberDS @andreas-junghanns 

Thank you.
 
**------------ Ready for Review -------------------**

**Review by andreas-junghanns — APPROVED** [review]
This is a HUGE step forward. Even if some minor issues remain to be fixed, this should be merged ASAP.
I would appreciate if a second reviewer approved as I was rather close to the effort and might not be impartial enough.

File: docs/5_2_scheduled_execution_api.adoc [hunk 1]
```diff
@@ -103,33 +106,31 @@ The <<fmi3ActivateModelPartition>> function has the following arguments:
...
 
-* [[clockElementIndex,`clockElementIndex`]] `clockElementIndex`: The element index of the <<Clock>> variable associated to the model partition which shall be activated.
-For a scalar <<Clock>> variable this must be 0; for array <<Clock>> variables, the element clock is specified using the 1-based element index.
+* [[clockElementIndex,`clockElementIndex`]] `clockElementIndex`: The element index of the <<Clock>> variable associated with the model partition which shall be activated.
+For a scalar <<Clock>> variable the index must be 0; for an array <<Clock>> variable, the index is 1-based.
 Using the element index 0 means all elements of the <<Clock>> variable.
 (Note: If an array has more than one dimension the indices are serialized in the same order as defined for values in <<serialization-of_variables>>).
 
 * [[activationTime, `activationTime`]] `activationTime`: value of the <<independent>> variable of the assigned <<Clock>> tick time latexmath:[\mathbf{t}_i] _[typically: simulation (i.e. virtual) time]_ (which is known to the simulation algorithm).
```
Comment (chrbertsch) [inline docs/5_2_scheduled_execution_api.adoc:114]: _[typically: simulation (i.e. virtual) time]_ 

I thought real-time applications are the most common ones, and virtual time is the exception

**Review by chrbertsch — APPROVED** [review]
This is really a major step forward. 

I just added a minor comment on simulated time vs. realtime; perhaps this aspect could be made more clear.

**Review by oliverkotte — APPROVED** [review]
The SE description is much clearer now, also for people like me who are non-natives in this field.

The perhaps improper naming of the state "Clock Activation Mode" has not been discussed. I point it out now and would leave it at that.

**Pull request merged**